ush/wave_outp_spec.sh

```diff
@@ -31,6 +31,11 @@ source "${USHgfs}/preamble.sh"
   workdir=$4
 
   YMDHE=$($NDATE $FHMAX_WAV_PNT $CDATE)
+  if [[ "${END_OF_IAU_START}" == "true" ]]; then
+    model_start_date=$(${NDATE} 3 ${CDATE})
+  else
+    model_start_date=${CDATE}
```

Comment:
We're trying to eliminate `CDATE`, so might as well make this correct from the start.
```suggestion
    model_start_date="${PDY}${cyc}"
```

Reply:
edited